DOCSP-37056 eloquent relationships (#2747)
jenssegers/laravel-mongodb: docs/eloquent-models.txt

.. facet::
   :name: genre
   :values: reference

.. meta::
   :keywords: php framework, odm

Eloquent models are part of the Laravel Eloquent object-relational

> mapping (ORM) framework that enable you to work with a database by using

Reviewer: S: the "that enable" is ambiguous 
```suggestion
Eloquent models are part of the Laravel Eloquent object-relational
mapping (ORM) framework. These models enable you to work with a database by using
```

Author: Good catch!
I'll use a slightly different structure than suggested since it's the framework rather than the models that relate to the second part.

> model classes. The {+odm-short+} extends this framework to use similar

Reviewer: ```suggestion
model classes. {+odm-short+} extends this framework to use similar
```

> syntax to work with MongoDB as a database.

Reviewer: Q: similar to what?

Author: I will rephrase to clarify which syntax. I wasn't able to find a canonical term to describe the syntax available in the Eloquent ORM, so I'll create a term, "Eloquent syntax".

This section contains guidance on how to use Eloquent models in
{+odm-short+} to work with MongoDB in the following ways:

> - :ref:`laravel-eloquent-model-relationships` shows how to add relationships

Reviewer: ```suggestion
- :ref:`laravel-eloquent-model-relationships` shows how to create relationships
```

Author: If the word choice seemed incorrect, I'll replace with "define" which is similar to what the Laravel docs use, e.g. https://laravel.com/docs/9.x/eloquent-relationships#defining-relationships
